src/App/ApplicationManager.py

```diff
@@ -250,7 +250,27 @@ def manage_minimize(self, value=1, REQUEST=None):
         self._getDB().cacheMinimize()
 
         if REQUEST is not None:
-            REQUEST.RESPONSE.redirect(REQUEST['URL1'] + '/manage_main')
+            msg = 'ZODB in-memory caches minimized.'
+            url = '%s/manage_main?manage_tabs_message=%s' % (REQUEST['URL1'],
+                                                             msg)
+            REQUEST.RESPONSE.redirect(url)
+
+    @requestmethod('POST')
+    def manage_pack(self, days=0, REQUEST=None, _when=None):
```

Comment:
Should this method be restricted to Manager users?

Reply:
It is because it has a `manage_` prefix. You should know this.

Reply:
I did not know that. It seems to happen in `InitializeClass`:
https://github.com/zopefoundation/AccessControl/blob/099c03fb5bcb7e3cf8e9f3f959a1a30508cd5422/src/AccessControl/class_init.py#L54-L57
Thank you for the hint.